.github/workflows/renovate-bot.yml

```diff
@@ -22,6 +22,7 @@ jobs:
                   # RENOVATE_TOKEN should have access :read_user, api, write_repository (repo checkbox)
                   RENOVATE_TOKEN: ${{ secrets.ROBOT_PAT }}
                   RENOVATE_CONFIG_FILE: ${{ github.workspace }}/renovate-config.js
+                  RENOVATE_GIT_AUTHOR: "Renovate Bot <renovate@whitesourcesoftware.com>"
```

Comment:
I think this is the Zowe Robot?

Reply:
Since the commit is done by renovate bot I used it as author.